ts/Extensions/Annotations/Types/Pitchfork.ts

```diff
@@ -39,11 +39,14 @@ if (defaultOptions.annotations) {
          * @optionparent annotations.types.pitchfork
          */
         {
+            /**
+             * @excluding line
+             */
             typeOptions: {
                 /**
                  * Inner background options.
                  *
-                 * @extends   annotations.types.crookedLine.shapeOptions
+                 * @extends   annotations.shapeOptions
```

Comment:
Could you elaborate? Pitchfork inherits from InfinityLine, which inherits from CrookedLine. I would rather expect `crookedLine` -> `InfinityLine` change? 🤔